test/test_charge_drift_models.jl

```diff
@@ -271,7 +271,7 @@ end
         @test cdm0.μ_e  == 0.1f0
         @test cdm0.μ_h == 0.1f0
 
-        cdm1 = IsotropicChargeDriftModel{T}(1000u"cm^2/(V*s)", 1000u"cm^2/(V*s)")
+        cdm1 = IsotropicChargeDriftModel{T}(μ_e = 1000u"cm^2/(V*s)", μ_h = 1000u"cm^2/(V*s)")
```

Comment:
I would like to also allow for the old syntax.
Can this be achieved using something like
```julia
IsotropicChargeDriftModel(μ_e::Union{RealQuantity, AbstractString}, μ_h::Union{RealQuantity, AbstractString}, args...; kwargs...) = IsotropicChargeDriftModel(args...; µ_e, µ_h, kwargs...)
```
?